frontend/src/components/NewGroup.tsx

```diff
@@ -70,14 +70,16 @@ function NewGroup(props: NewGroupProps) {
   const [friends, setFriends] = React.useState<Friend[]>([user]);
   const [splitMode, setSplitMode] = useState<"equally" | "byAmount">("equally");
   const today = new Date();
+  const next_month: Date = new Date(today.getTime()); // Cloning today's date
+  next_month.setDate(today.getDate() + 30);
 
   const subscriptions = [
-    new Subscription("Netflix", netflixImage, 20, today),
-    new Subscription("HBO", hboImage, 30, today),
-    new Subscription("Disney", disney, 25, today),
-    new Subscription("Spotify", spotify, 10, today),
-    new Subscription("Youtube", youtubeImage, 12, today),
-    new Subscription("Crunchy", crunchyrollImage, 7, today),
+    new Subscription("Netflix", netflixImage, 20, today, next_month),
```

Comment:
set the next_billing_date from the server side, instead of the client side here. Otherwise, the user can send whatever billing date to the API, but this shouldn't be part of the user input.